parlai/core/torch_agent.py

```diff
@@ -391,6 +391,19 @@ def history_class(cls):
     def add_cmdline_args(cls, argparser):
         """Add the default commandline args we expect most agents to want."""
         agent = argparser.add_argument_group('TorchAgent Arguments')
+        agent.add_argument(
+            '-i',
+            '--interactive-mode',
+            type='bool',
+            default=True,
+            help='Whether in full interactive mode or not,  which means generating text or '
+            ' retrieving from a full set of candidates, which is necessary to actually '
```

Comment:
nit: you have spaces on each end of the line

Reply:
i thought we weren't doing lint. and i ran the black thing.. and you're STILL on my back?? omg omg omg

Reply:
I think she means that the strings concatenate, so you've written double spaces everywhere. (i.e. a space after "or" and before "retrieving")